src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -185,12 +185,18 @@ export default class WorkspaceNotebooks extends Component {
   loadClusters() {
     const { namespace } = this.props.workspace
 
-    this.setState({ clusters: undefined })
+    this.setState({ clusters: undefined, cluster: undefined })
     Leo.clustersList().then(
       list => {
+        const cluster = _.find(list,
+          c => c.googleProject === namespace &&
+            c.creator === Utils.getUser().getBasicProfile().getEmail())
+        if (cluster) {
+          Leo.notebooks(namespace, cluster.clusterName).setCookie().then(this.setState({ cluster }))
```

Comment:
This doesn't look right. Was this supposed to be `.then(() => this.setState({ cluster }))`?